tools/llvm-spirv/llvm-spirv.cpp

```diff
@@ -139,7 +139,7 @@ static int convertLLVMToSPIRV() {
   std::string Err;
   bool Success = false;
   if (OutputFile != "-") {
-    std::ofstream OutFile(OutputFile);
+    std::ofstream OutFile(OutputFile, std::ios::binary);
```

Comment:
Whether we provide `std::ios::binary` should perhaps be conditional on `SPIRV::SPIRVUseTextFormat`, see line 136?
(although I am not entirely sure when `SPIRVUseTextFormat` would be true in normal usage of llvm-spirv)

Reply:
I'm able to generate readable text output using `std::ios::binary`, but it'd be easy to chose the file mode selectively if desired.  Something like:

````c++
    std::ofstream OutFile;
    if (SPIRV::SPIRVUseTextFormat) {
      OutFile.open(OutputFile);
    } else {
      OutFile.open(OutputFile, std::ios::binary);
    }
````

Let me know what you'd prefer, thanks!

Reply:
> I'm able to generate readable text output using `std::ios::binary`

Fair enough, then your patch should be fine, thanks!